### PR TITLE
Update it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -1065,18 +1028,21 @@ msgid ""
 "closes. This will allow Planner to continue updating and sending reminder "
 "notifications."
 msgstr ""
+"Abilita questa impostazione per mantenere Planner in esecuzione in background quando "
+"chiude. Ciò consentirà a Planner di continuare ad aggiornare e inviare notifiche di "
+"promemoria".
```
Comment (alainm23) [inline po/it.po]: syntax error

File: po/it.po [hunk 2]
```diff
@@ -1194,21 +1157,23 @@ msgid ""
...
+"ti piace Planner e vuoi supportarne lo sviluppo, considera di donare a"
+"attraverso:"
 
 #: src/Dialogs/Preferences/Preferences.vala:1150
 msgid ""
 "Thanks to them who made a donation via Patreon or PayPal. (If you want to "
 "appear here visit our Patreon account 😉️)"
 msgstr ""
+"Grazie a loro che hanno fatto una donazione tramite Patreon o PayPal. (Se vuoi"
+"apparire qui, visita il nostro account Patreon 😉️)
```
Comment (alainm23) [inline po/it.po]: end-of-line within string

**Review by alainm23 — CHANGES_REQUESTED** [review]
Thank you very much for helping to translate Planner. There are some syntax errors. Please fix it.

**Review by albanobattistella — COMMENTED** [review]
fixed.

**Pull request merged**